proxy/src/serve_sockets.rs

```diff
@@ -466,20 +464,35 @@ mod tests {
     async fn test_encryption() {
         let mut key = GenericArray::default();
         OsRng.fill_bytes(&mut key);
-        const N: usize = 2_usize.pow(13);
-        let test_data: &mut [u8; N] = &mut [0; N];
-        OsRng.fill_bytes(test_data);
-        let mut read_buf = [0; N];
+        let data: Arc<Vec<_>> = (0..13337).map(|_| {
+            let mut chunk = vec![0; OsRng.gen_range(1..9999)];
```

Comment:
9999 because its larger than 8192 which is the default buffer size of `tokio::io::copy_bidirectional` and 13337 because its a cool number and approximately the number of writes that SEL did in my test case.